.github/workflows/python.yml

```
@@ -56,7 +56,7 @@ jobs:
         run: pip -v install dist/*.whl
 
       - name: Test import
-        run: python -c 'import idyntree.bindings'
+        run: python -c 'import idyntree.swig'
```

```suggestion
        run: python -c 'import idyntree.bindings'
```
See previous comment.

---

Done